tests/playbooks/tests_ethtool_features.yml

```diff
@@ -145,6 +145,10 @@
                       features:
                         tx_tcp_segmentation: "no"
                         tx-tcp-segmentation: "no"
+                ansible_managed_new: "{{ __network_new_header }}"
+              vars:
+                __network_new_header: "{{
+                  lookup('template', 'get_ansible_managed.j2') }}"
```

Comment:
What is the  purpose of only adding this to `tests/playbooks/tests_ethtool_features.yml` and `tests/playbooks/tests_reapply.yml` ? For passing the integration test ?

Reply:
> What is the purpose of only adding this to `tests/playbooks/tests_ethtool_features.yml` and `tests/playbooks/tests_reapply.yml` ? For passing the integration test ?

Yes. Without adding them, tests including the playbooks fail. You wrote "only adding to ...". Do you think I missed adding the ansible managed parameter at any other places where I should have? Thanks!

Reply:
No, you are not missing adding it somewhere else.